sirepo/sim_data/flash.py

```diff
@@ -22,10 +23,13 @@ def fixup_old_data(cls, data):
         dm = data.models
         for m in list(dm.keys()):
             n = m.replace(':', '').replace('magnetoHD', '')
+            n = re.sub('CapLaser$', 'CapLaserBELLA', n)
```

Comment:
Something is wrong here. The m != n test should be before this line and/or needs to be done for this line as well.

Reply:
Can you give an example of what problem you see? I'm not seeing it.

Reply:
OK, I see the new case, which is confusing so perhaps this would be clearer:

```txt
n = re.sub(
    'CapLaser$', 
    'CapLaserBELLA', 
    m.replace(':', '').replace('magnetoHD', ''),
)
```

Seeing the difference between `m` and `n` on line 26 was confusing to me.  Perhaps you could iterate the transforms (they could all be regexes):

```py
n = m
for x in (':', ''), ('magnetoHD', ...)...:
    n = re.sub(x[0], x[1], n)
```

Reply:
Done https://github.com/radiasoft/sirepo/commit/7abf69f208caf462152e0803f71d0d6c7996fb35